doc/mainpage.dox

```diff
@@ -162,8 +162,18 @@ The `D_xy` tag takes the following arguments through sub-tags:
 
 \include tests/serverTestXy-DoocsVariableConfig.xml
 
+\subsubsection set_error set_error
```

Comment:
Why is this a subsubsection of "Special DOOCS properties"? I would make it a subsection (i.e. on the same level as "Special DOOCS properties") called "Error reporting" or so.

Reply:
That's because it affects the special DOOCS properties for error reporting. But I'm also fine with your suggestion. Changed it.